apps/evaluations/tables.py

```diff
@@ -216,13 +216,22 @@ class Meta:
 
 
 class EvaluationSessionsSelectionTable(tables.Table):
-    selection = columns.CheckBoxColumn(
-        accessor="external_id",
-        verbose_name="Select",
+    selection = TemplateColumn(
+        template_name="evaluations/session_checkbox.html",
+        verbose_name="All Messages",
         orderable=False,
-        attrs={
-            "input": {"class": "checkbox checkbox-primary session-checkbox", "@change": "updateSelectedSessions()"},
-            "th__input": {"style": "display: none;"},  # Hide the select all checkbox in header
+        extra_context={
+            "css_class": "checkbox checkbox-primary session-checkbox",
+            "js_function": "updateSelectedSessions()",
+        },
+    )
+    clone_filtered_only = TemplateColumn(
```

Comment:
Does it make sense to configure this per session, or should it not be configured once for all sessions? Also, what if both `All Messages` and `Filtered Messages` are selected?

Reply:
Good question! I interpreted the spec

> User can import ALL messages from the selected sessions OR only messages matching the tag filter.

to be selectable for each session, but I can see how you could interpret it as a switch on the whole cloning operation. Maybe @snopoke has opinions? If the switch is at the clone level, it does remove a lot of complexity here. 

Also, good catch - I'll update to only allow a single checkbox to be checked. I forgot to add that.